### PR TITLE
Add JACS CKD2 inputs

### PR DESCRIPTION
**Pull request by IAlibay**
## Input submission checklist

* [x] created a directory for each system based on the [template directory][templates]
* [x] named and placed the directory with the following pattern: `input_structures/prepared_structures/<set_name>/<system_name>`

For each submitted directory:

* [x] filled in system preparation details in the `PREPARATION_DETAILS.md` file
* [x] added a PDB file with the protein named `protein.pdb`
* [x] removed any cofactors from the PDB file and placed them in `cofactors.sdf`
* [ ] did not re-protonate the residues, but left protonation state of the protein unchanged
* [x] kept cystallographic waters and metals in the PDB file
* [x] tested the PDB and, if available, cofactors.sdf file using the [input validation script][input_validation]
* [x] copied over the ligands SDF file to a file named `ligands.sdf` and removed any duplicate binding modes or protonation states, keeping the more favorable state

## Licensing agreement
* [x] I agree to release these inputs under the combined MIT and CCBY SA 4.0 licenses of this repository

[templates]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks
[input_validation]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks/utils/input_validation.py

### CODE REVIEW

File: industry_benchmarks/input_structures/prepared_structures/jacs_set/cdk2/protein.pdb
Comment (IAlibay) [inline industry_benchmarks/input_structures/prepared_structures/jacs_set/cdk2/protein.pdb:1]: The one thing I did notice was that the protons seems to have been re-added, there's no significant changes, but the steps seems to have re-assigned everything.

File: industry_benchmarks/input_structures/prepared_structures/jacs_set/tyk2/ligands.sdf
Comment (IAlibay) [inline industry_benchmarks/input_structures/prepared_structures/jacs_set/tyk2/ligands.sdf:1]: This is just a permissions change on the file, it had executable permissions on so I did `chmod -x` on it.

File: industry_benchmarks/input_structures/prepared_structures/jacs_set/cdk2/protein.pdb
Comment (hannahbaumann) [reply industry_benchmarks/input_structures/prepared_structures/jacs_set/cdk2/protein.pdb:1]: Should the two NMA residues be renamed to NME?

**Review by hannahbaumann — CHANGES_REQUESTED** [review]
Just the one comment, otherwise looks good to me!

**Comment by IAlibay** [comment]
@hannahbaumann caps have been renamed & resids fixed

**Review by hannahbaumann — APPROVED** [review]
Thanks, lgtm!

**Pull request merged**